build.sbt

```diff
@@ -0,0 +1,58 @@
+import ReleaseTransformations._
+
+name := "commercetools-sunrise-email"
+
+organization := "io.commercetools.sunrise"
+
+lazy val `commercetools-sunrise-email` = (project in file("."))
+  .configs(IntegrationTest)
+  .settings(commonSettings ++ commonTestSettings : _*)
+
+lazy val commonSettings = releaseSettings ++ Seq (
+  scalaVersion := "2.11.7",
+  javacOptions ++= Seq("-source", "1.8", "-target", "1.8")
+)
+
+javaUnidocSettings
+
+/**
+ * TEST SETTINGS
+ */
+
+lazy val commonTestSettings = itBaseTestSettings ++ configCommonTestSettings("test,it")
+
+lazy val itBaseTestSettings = Defaults.itSettings ++ configTestDirs(IntegrationTest, "it")
+
+def configTestDirs(config: Configuration, folderName: String) = Seq(
+  javaSource in config := baseDirectory.value / folderName,
+  scalaSource in config := baseDirectory.value / folderName,
+  resourceDirectory in config := baseDirectory.value / s"$folderName/resources"
+)
+
+def configCommonTestSettings(scopes: String) = Seq(
+  testOptions += Tests.Argument(TestFrameworks.JUnit, "-v"),
+  libraryDependencies ++= Seq (
+    "org.assertj" % "assertj-core" % "3.0.0" % scopes
+  ),
+  dependencyOverrides ++= Set (
+    "junit" % "junit" % "4.12" % scopes
```

Comment:
can you add two dummy tests for test and it? Sometimes SBT is bitchy concerning runnning jUnit tests. I expected this as dependency: https://github.com/sbt/junit-interface

http://www.scala-sbt.org/0.13/docs/Testing.html#JUnit